Panic in AWS secrets engine when generating OpenAPI

I have noticed that if I run `./scripts/gen_openapi.sh` I receive this panic:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xf20487]

goroutine 458 [running]:
github.com/hashicorp/vault/builtin/credential/aws.(*backend).upgrade(0xc00070a0b0, 0x3781680, 0xc000224540, 0x0, 0x0, 0xc000758700, 0x3f7799e520000000, 0x3f7799e520000000)
	/home/tbex/go/src/github.com/hashicorp/vault/builtin/credential/aws/path_role.go:379 +0x37
github.com/hashicorp/vault/builtin/credential/aws.(*backend).initialize.func1(0xc00070a0b0, 0x3781680, 0xc000224540, 0x0, 0x0, 0x37ceb60, 0xc0000a2960)
	/home/tbex/go/src/github.com/hashicorp/vault/builtin/credential/aws/path_role.go:352 +0xa0
created by github.com/hashicorp/vault/builtin/credential/aws.(*backend).initialize
	/home/tbex/go/src/github.com/hashicorp/vault/builtin/credential/aws/path_role.go:343 +0x1b7
```
I'm currently poking at the issue. We've worked on that area of code lately so I want to document this just in case it might indicate an issue with the beta we're currently in.

--- a/vault/logical_system.go
+++ b/vault/logical_system.go
@@ -3250,6 +3250,7 @@ func (b *SystemBackend) pathInternalOpenAPI(ctx context.Context, req *logical.Re
 
 			req := &logical.Request{
 				Operation: logical.HelpOperation,
+				Storage: req.Storage,
 			}
 
 			resp, err := backend.HandleRequest(ctx, req)

--- a/vault/logical_system.go
+++ b/vault/logical_system.go
@@ -3250,7 +3250,7 @@ func (b *SystemBackend) pathInternalOpenAPI(ctx context.Context, req *logical.Re
 
 			req := &logical.Request{
 				Operation: logical.HelpOperation,
-				Storage: req.Storage,
+				Storage:   req.Storage,
 			}
 
 			resp, err := backend.HandleRequest(ctx, req)

